mentat/vision/vision_manager.py

```diff
@@ -29,7 +29,7 @@ def open(self, path: str) -> None:
 
     def screenshot(self, path: Optional[str] = None) -> str:
         if path is not None:
-            expanded = os.path.expanduser(path)
+            expanded = os.path.abspath(path)
```

Comment:
Good catch! I think we should have both. As is `/screenshot ~/.mentat/conversation_viewer.html` won't work.

Reply:
Oh I didn't realize abspath didn't do that, thanks! I'll do that